pkg/controller/appdefinition/volume.go

```diff
@@ -197,16 +196,7 @@ func toPVCs(req router.Request, appInstance *v1.AppInstance) (result []kclient.O
 			pvc.Spec.VolumeName = pvName
 
 			if volumeRequest.Size == "" {
-				// If the volumeRequest does not have a size set, then we need to determine what default to use. If status.Defaults.Volumes has this
-				// volume set, then we use that. Otherwise, we use v1 package level default size.
-				defaultSize := *v1.DefaultSize
-				if defaultVolume, hasDefaultSet := appInstance.Status.Defaults.Volumes[vol]; hasDefaultSet {
-					defaultSize, err = resource.ParseQuantity(string(defaultVolume.Size))
-					if err != nil {
-						return nil, fmt.Errorf("failed to parse default volume size %q for volume %q: %w", defaultVolume.Size, vol, err)
-					}
-				}
-				pvc.Spec.Resources.Requests[corev1.ResourceStorage] = defaultSize
+				pvc.Spec.Resources.Requests[corev1.ResourceStorage] = *v1.DefaultSize
```

Comment:
Not sure why I removed that, lol. I'll add it back.

Reply:
Oh wait, nvm. The reason I removed that is because the refactored ResolveVolumeRequest function (called earlier here) will handle setting the default size and checking to see if there is already a size that was previously resolved. So by the time we get down here, if there is still no determined size, the only thing left to use is `v1.DefaultSize`.